### DISCUSSION TITLE
What's the difference in the two play buttons

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [dvmarinoff/Auuki#134] [Q&A] [answered]
Hi - I looked around here but I don't see what the difference is in these buttons?

![image](https://user-images.githubusercontent.com/68966864/196010290-8f44ea15-07e3-48da-9df3-f51ccd98ec5e.png)

**Comment by dvmarinoff** [comment, marked as answer]
Hi @htims05,

The play button WITH a circle starts both Recording and the execution of the chosen Workout.
The plain play button starts only Recording and NOT the Workout. This way the User can later chose to, or not to start a workout.

**Reply by Lunchtime0614** [reply]
Thanks!

**Comment by Thibaultfq** [comment]
Although i think this is a great feature, I do not think that these icons are intuitive. I "accidently" pressed the start timer because i had no idea about the difference of the icons and was wondering why my laps were not progressing.

a rule of thumb when using icons only: only use if you are 100% sure every user knows exactly what they mean, otherwise complement them with text.

suggestion here: complement with 'start workout and recording' and 'start recording only' . May be not be as clean in the UI but avoids any confusion.

**Reply by dvmarinoff** [reply]
Sorry for your frustration with the current design. At the begging I planned to add a ‘guided tour’ kind of feature for new users, but never got enough time to go and implementing it.
